src/nft/hooks/useWalletCollections.ts

```diff
@@ -38,7 +38,7 @@ export const useWalletCollections = create<WalletCollectionState>()(
       setCollectionFilters: (address) =>
         set(({ collectionFilters }) => {
           if (collectionFilters.length === 0) return { collectionFilters: [address] }
-          else if (!!collectionFilters.find((x) => x === address))
+          else if (collectionFilters.find((x) => x === address))
```

Comment:
`find` is incorrect - use [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)